fetch.bs

```diff
@@ -4792,7 +4792,11 @@ steps:
  <a for="fetch params">process response</a> given <var>response</var>, with <var>fetchParams</var>'s
  <a for="fetch params">task destination</a>.
 
- <li><p>If <var>response</var>'s <a for=response>body</a> is null, then run
+ <li><p>Let <var>internalResponse</var> be <var>response</var>, if <var>response</var> is a
+ <a>network error</a>; otherwise <var>response</var>'s
+ <a for="filtered response">internal response</a> otherwise.
```

Comment:
Drop "otherwise" at the end here.

Also not entirely sure we need the comma before "if". I think we usually leave it out if the sentences are relatively short.

Reply:
Done (oops)